qcore/test/test_source_site_dist/test_src_site_dist_calc.py

```diff
@@ -1,13 +1,64 @@
 import os
 import pickle
+import shutil
+import sys
 
 import numpy as np
 import pytest
 
 from qcore import geo, src_site_dist as ssd
+from qcore import shared
 
 INPUT = "input"
 OUTPUT = "output"
+REALISATIONS = [
+    (
+        "PangopangoF29_HYP01-10_S1244",
+        "https://qc-s3-autotest.s3-ap-southeast-2.amazonaws.com/testing/IM_calculation/PangopangoF29_HYP01-10_S1244.zip",
+    )
+]
+
+
+@pytest.yield_fixture(scope="session", autouse=True)
+def set_up(request):
+    test_data_save_dirs = []
+    for i, (REALISATION, DATA_DOWNLOAD_PATH) in enumerate(REALISATIONS):
+        data_store_path = os.path.join(os.getcwd(), "sample" + str(i))
+        zip_download_path = os.path.join(data_store_path, REALISATION + ".zip")
+
+        download_cmd = "wget -O {} {}".format(zip_download_path, DATA_DOWNLOAD_PATH)
+        unzip_cmd = "unzip {} -d {}".format(zip_download_path, data_store_path)
+        # print(DATA_STORE_PATH)
+        test_data_save_dirs.append(data_store_path)
+        if not os.path.isdir(data_store_path):
+            os.makedirs(data_store_path, exist_ok=True)
+            out, err = shared.exe(download_cmd, debug=False)
+            if "error" in err:
+                shutil.rmtree(data_store_path)
+                sys.exit("{} failed to retrieve test data".format(err))
+            # download_via_ftp(DATA_DOWNLOAD_PATH, zip_download_path)
```

Comment:
do we really need the commented lines here? We can always fish them out of the git history if we need them.